.github/workflows/learninh-github-actions.yml

```diff
@@ -0,0 +1,13 @@
+name: Github Actions test
+run-name: ${{ github.actor }} is leaarning GitHub Actions 🚀
+on: [push]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '^14'
+      - run: npm install -g bats
```

Comment:
Bats? Why do you need this?

Reply:
it was just a example that was in github docs and i wanted see what would happens using this.

Reply:
I removed it because the npm test was already running and just to cover this test, bats is not necessary